code/game/objects/items/devices/flashlight.dm

```diff
@@ -762,7 +762,7 @@
 	var/fuel = 0
 
 /obj/item/flashlight/glowstick/Initialize(mapload)
-	fuel = rand(50 MINUTES, 60 MINUTES)
+	fuel = rand(10 MINUTES, 15 MINUTES)
```

Comment:
Maybe 18-20 minutes, or 20-25 if you feel like spending a couple minutes making it so their range wanes with time.

Reply:
I'll buy that, sure